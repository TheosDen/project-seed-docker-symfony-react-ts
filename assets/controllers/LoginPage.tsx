import React = require('react');
import { Button, Form } from 'react-bootstrap'

export default function LoginPage () {
  return <Form>
        <Form.Group controlId="formBasicEmail">
            <Form.Label>Login</Form.Label>
            <Form.Control type="text" placeholder="Enter login" />
        </Form.Group>

        <Form.Group controlId="formBasicPassword">
            <Form.Label>Password</Form.Label>
            <Form.Control type="password" placeholder="Password" />
        </Form.Group>

        <Form.Group controlId="formBasicCheckbox">
            <Form.Check type="checkbox" label="Remember me" />
        </Form.Group>

        <Button variant="primary" type="submit">
            Submit
        </Button>
    </Form>
}
