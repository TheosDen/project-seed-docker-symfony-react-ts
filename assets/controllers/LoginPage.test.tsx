import * as React from 'react'
import * as renderer from 'react-test-renderer'
import LoginPage from './LoginPage'

it('LoginPage renders correctly', () => {
  const tree = renderer
    .create(<LoginPage />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
