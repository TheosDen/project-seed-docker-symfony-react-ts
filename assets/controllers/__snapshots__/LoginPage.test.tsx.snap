// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage renders correctly 1`] = `
<form
  className=""
>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="formBasicEmail"
    >
      Login
    </label>
    <input
      className="form-control"
      id="formBasicEmail"
      placeholder="Enter login"
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="form-label"
      htmlFor="formBasicPassword"
    >
      Password
    </label>
    <input
      className="form-control"
      id="formBasicPassword"
      placeholder="Password"
      type="password"
    />
  </div>
  <div
    className="form-group"
  >
    <div
      className="form-check"
    >
      <input
        className="form-check-input"
        disabled={false}
        id="formBasicCheckbox"
        type="checkbox"
      />
      <label
        className="form-check-label"
        htmlFor="formBasicCheckbox"
        title=""
      >
        Remember me
      </label>
    </div>
  </div>
  <button
    className="btn btn-primary"
    disabled={false}
    type="submit"
  >
    Submit
  </button>
</form>
`;
